### PR TITLE
Enable publishing on crates.io

### PR DESCRIPTION
**Pull request by bruno-delfino1995**
As the crates had names that would clash with existing crates on
`crates.io`, I've changed their names to contain the project name as a
prefix, which bypasses the clash and allows us to publish there. Also,
I've changed the release workflow to publish at crates.io along github
releases.

Close: #4

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -48,23 +47,13 @@ jobs:
...
-            rust: nightly
-            target: x86_64-pc-windows-msvc
-            bin: kct.exe
-            name: kct-windows-amd64.exe
-          - os: macOS-latest
-            rust: nightly
-            target: x86_64-apple-darwin
-            bin: kct
-            name: kct-darwin-amd64
     runs-on: ${{ matrix.os }}
```
Comment (thepabloaguilar) [inline .github/workflows/release.yml:57]: You don't need to use a matrix strategy here, you just have one `os`!
And you can set `rust`, `target`, `bin` and `name` as environment variable to use inside your workflow.

**Review by thepabloaguilar — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (bruno-delfino1995) [reply .github/workflows/release.yml:57]: I 100% agree and have kept it purely out of laziness with the hopes of building for multiple platforms someday.

I will remove it.

Comment (thepabloaguilar) [reply .github/workflows/release.yml:57]: Well, you don't need to use the `workflow` commands here, you just use them if you want to get an output of any step to use in another step!

You can set env variables for all steps, see the documentation [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idenv)

Comment (bruno-delfino1995) [reply .github/workflows/release.yml]: I wasn't aware of those. Thanks for bringing it in.

**Review by thepabloaguilar — APPROVED** [review]

**Pull request merged**